src/components/NewStudentForm/form.js

```diff
@@ -39,6 +39,7 @@ class StudentInfoForm extends PureComponent<Props, State> {
   };
 
   state: State = Object.assign({...this.defaultFields}, {
+    emailOptIn: true,
```

Comment:
I think we'll need to account for if the student already has the emailOptOut unchecked as well